PolyEngine/RenderingDevice/OpenGL/Src/TiledForwardRenderer.cpp

```diff
@@ -1312,29 +1312,42 @@ void TiledForwardRenderer::PostGamma(const SceneView& sceneView)
 {
 	float time = (float)TimeSystem::GetTimerElapsedTime(sceneView.WorldData, eEngineTimer::GAMEPLAY);
 
+	float planeNear = sceneView.CameraCmp->GetClippingPlaneNear();
+	float planeFar = sceneView.CameraCmp->GetClippingPlaneFar();
+
 	const ScreenSize screenSize = RDI->GetScreenSize();
 
 	float grainScale = 0.01f;
 	float vignetteScale = 0.1f;
 	float abberationScale = 0.1f;
 	Color tint = Color::WHITE;
+	Color fogColor = Color::WHITE;
+	float fogDensity = 0.66f;
```

Comment:
Same here. Also why is this duplicated?

Reply:
These are default values in case when PostProcessSettingsComponent is not present. Should be moved to config soon.